arm-resources/resources/2016-07-01/swagger/resources.json

```diff
@@ -2459,9 +2459,6 @@
           "description": "Resource tags"
         }
       },
```

Comment:
@vivsriaus - Usually we do not generate clients from the old versions. That has been the model till now. If you have to bug fix a client for an old api-version then these changes would go along with the bug fix.

As a general rule, it is always good to keep the swagger spec in a good condition that is accurately and correctly describes the swagger spec.